[32mINFO    [0m root:test_task_post.py:23 Executing Incomplete Task API
[32mINFO    [0m root:test_task_post.py:24 Sending the POST request with task id oIIpMGzt
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:226 Starting new HTTP connection (1): localhost:5000
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:433 http://localhost:5000 "POST /tasks/oIIpMGzt/incomplete HTTP/1.1" 200 2
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:226 Starting new HTTP connection (1): localhost:5000
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:433 http://localhost:5000 "GET /tasks/oIIpMGzt HTTP/1.1" 200 39
[32mINFO    [0m root:test_task_post.py:26 Task status completed False